### ISSUE TITLE
"Stop Recording" button should be shown on page reload if recording is active

### ISSUE THREAD

**Issue by leigh-johnson** [bitsy-ai/printnanny-os#271]
## **Describe the bug**
The "Start Recording" button is incorrectly shown when a recording is already in progress.

## **To Reproduce**
1. Browse to http://travelpi.local/cam
2. Click "Start Recording"
3. Refresh the page
4. Notice: "Start Recording" button is shown, even though a recording is already in progress.

## **Expected behavior**
This button should say "stop recording" when a recording is in-progress.

## **Screenshots**
<img width="993" alt="Screenshot 2023-02-16 at 1 22 25 PM" src="https://user-images.githubusercontent.com/2601819/219364054-64288f9d-fb79-4ac9-a7f7-5fed94b74f79.png">

## Issue.txt
```
-----------------------
Build Info:  |
-----------------------
IMAGE_NAME = printnanny-release-image-raspberrypi4-64-20230216105936
DISTRO_PRETTY = PrintNanny Linux 0.6.5 (Feldspar)
VARIANT_NAME = PrintNanny OS Core
VARIANT_ID = core
DISTRO = printnanny
DISTRO_NAME = PrintNanny Linux
DISTRO_VERSION = 0.6.5
DISTRO_CODENAME = Feldspar
ID_LIKE = BitsyLinux
HOME_URL = https://printnanny.ai
BUG_REPORT_URL = https://github.com/bitsy-ai/printnanny-os/issues
YOCTO_VERSION = 4.1.2
YOCTO_CODENAME = Langdale
MACHINE = raspberrypi4-64
TUNE_PKGARCH = cortexa72
-----------------------
Layer Revisions:      |
-----------------------
meta              = langdale:f59aa3752dbc5552469207894bea9c01bd68fb6e
meta-poky         = langdale:f59aa3752dbc5552469207894bea9c01bd68fb6e
meta-yocto-bsp    = langdale:f59aa3752dbc5552469207894bea9c01bd68fb6e
meta-raspberrypi  = langdale-libcamera-fix:99f5a91d9d4b34ff8c214cf61cd12f375da1a76a
meta-oe           = langdale:f8cb46d803190bb02085c8a7d20957a71d32f311
meta-python       = langdale:f8cb46d803190bb02085c8a7d20957a71d32f311
meta-multimedia   = langdale:f8cb46d803190bb02085c8a7d20957a71d32f311
meta-bitsy        = main:b7aa872b3fd6ed6f976698c7bd61dfdc5b4e5527
meta-neural-network = mickledore:f19fa74c2f4c868eda1c48d709b66848f9cca5e8
meta-printnanny   = main:b7aa872b3fd6ed6f976698c7bd61dfdc5b4e5527
meta-networking   = langdale:f8cb46d803190bb02085c8a7d20957a71d32f311
meta-filesystems  = langdale:f8cb46d803190bb02085c8a7d20957a71d32f311
meta-initramfs    = langdale:f8cb46d803190bb02085c8a7d20957a71d32f311
meta-webserver    = langdale:f8cb46d803190bb02085c8a7d20957a71d32f311
meta-swupdate     = mickledore:c47a10be18412919f4c7041be05509ba97ac79f4
meta-microcontroller = mickledore:a848733698649628d0c3e34ae04ced0ee6ce3aee
```

**Comment by leigh-johnson** [comment]
Re-opening as a regression, seeing reports of this in the 0.6.5 release build.

**Comment by leigh-johnson** [comment]
Confirming this is fixed in v0.6.6 nightly. Bonus corgi footage from my travel Raspberry Pi:


<img width="1138" alt="Screenshot 2023-02-27 at 7 39 20 PM" src="https://user-images.githubusercontent.com/2601819/221653676-8f8f2324-5ca5-46da-b2bd-e20805e878ef.png">
